build/pipelines/templates-v2/pipeline-onebranch-full-release-build.yml

```diff
@@ -0,0 +1,256 @@
+    ref: refs/heads/main
+
+extends:
+  ${{ if eq(parameters.official, true) }}:
+    template: v2/Microsoft.Official.yml@templates # https://aka.ms/obpipelines/templates
+  ${{ else }}:
+    template: v2/Microsoft.NonOfficial.yml@templates
+  parameters:
+    featureFlags:
+      WindowsHostVersion: 1ESWindows2022
+    platform:
+      name: 'windows_undocked'
+      product: 'Windows Terminal'
+    cloudvault: # https://aka.ms/obpipelines/cloudvault
+      enabled: false
+    globalSdl: # https://aka.ms/obpipelines/sdl
+      tsa:
+        enabled: true
+        configFile: '$(Build.SourcesDirectory)\build\config\tsa.json'
+      binskim:
+        break: false
+        scanOutputDirectoryOnly: true
+      policheck:
+        break: false
+        severity: Note
+      baseline:
+        baselineFile: '$(Build.SourcesDirectory)\build\config\release.gdnbaselines'
+        suppressionSet: default
+
+    stages:
+      - stage: Build
+        displayName: Build
+        dependsOn: []
+        jobs:
+          - template: ./build/pipelines/templates-v2/job-build-project.yml@self
+            parameters:
+              pool: { type: windows }
+              variables:
+                ob_git_checkout: false # This job checks itself out
+                ob_git_skip_checkout_none: true
+                ob_outputDirectory: $(JobOutputDirectory)
+                ob_artifactBaseName: $(JobOutputArtifactName)
+              publishArtifacts: false # Handled by OneBranch
+              branding: ${{ parameters.branding }}
+              buildTerminal: ${{ parameters.buildTerminal }}
+              buildConPTY: ${{ parameters.buildConPTY }}
+              buildWPF: ${{ parameters.buildWPF }}
+              pgoBuildMode: ${{ parameters.pgoBuildMode }}
+              buildConfigurations: ${{ parameters.buildConfigurations }}
+              buildPlatforms: ${{ parameters.buildPlatforms }}
+              generateSbom: false # this is handled by onebranch
+              removeAllNonSignedFiles: true # appease the overlords
+              codeSign: ${{ parameters.codeSign }}
+              beforeBuildSteps: # Right before we build, lay down the universal package and localizations
+                - task: PkgESSetupBuild@12
+                  displayName: Package ES - Setup Build
+                  inputs:
+                    disableOutputRedirect: true
+
+                - task: UniversalPackages@0
+                  displayName: Download terminal-internal Universal Package
+                  inputs:
+                    feedListDownload: 2b3f8893-a6e8-411f-b197-a9e05576da48
+                    packageListDownload: e82d490c-af86-4733-9dc4-07b772033204
+                    versionListDownload: ${{ parameters.terminalInternalPackageVersion }}
+
+                - template: ./build/pipelines/templates-v2/steps-fetch-and-prepare-localizations.yml@self
+                  parameters:
+                    includePseudoLoc: true

```

Comment:
There's some duplication here between the non-OB and OB release builds.

However: we can put the non-OB release builds on a path to deprecation.